services/core-api/app/api/now_applications/response_models.py

```diff
@@ -801,6 +801,68 @@ def format(self, value):
         fields.String,
     })
 
+NOW_VIEW_MODEL_PROPONENT = api.model(
+    'NOW_VIEW_MODEL_PROPONENT', {
+        'now_application_guid':
+        fields.String,
+        'mine_guid':
+        fields.String,
+        'mine_no':
+        fields.String,
+        'mine_name':
+        fields.String,
+        'mine_region':
+        fields.String,
+        'now_number':
+        fields.String,
+        'permit_no':
+        fields.String,
+        'party':
+        fields.String,
+        'lead_inspector_name':
```

Comment:
wasn't the lead inspector name something that isn't supposed to go through? Could probably limit the fields to what's shown on the table for now and can always add them later if they're needed